src/components/themes/themes.js

```diff
@@ -93,6 +93,15 @@ const themes = {
       color: '#cf1c61',
       borderColor: '#fd8ac0'
     }
+  },
+  pricing: {
+    image: <Image icon="cart" color="green" />,
```

Comment:
@HeyStenson What do you think about `creditcard`?

![image](https://user-images.githubusercontent.com/2180540/110333979-41568580-7ff0-11eb-93af-6366342d6c6d.png)

Reply:
Much better! I didn't see that one, probably bc I was too fixated on finding 💸  😆

Reply:
wow this is so great!